Formula/atasm.rb

```diff
@@ -23,7 +28,7 @@ def install
       system "sed -e 's,%%DOCDIR%%,/usr/local/share/doc/atasm,g' < atasm.1.in > atasm.1"
```

Comment:
Can we do an `inreplace` here instead? `/usr/local` should probably also be `HOMEBREW_PREFIX`, I suppose.